src/components/adjudications/AdjudicationsSection.jsx

```diff
@@ -12,12 +12,19 @@ class AdjudicationsSection extends React.Component {
 
     this.state = {
       adjudications: [],
-      loading: true
+      loading: true,
+      performanceValues: {}
     };
   }
 
   componentDidMount() {
     const { match: { params: { eventId, performanceId }}} = this.props;
+
+    const docRef = db.collection('events').doc(eventId).collection('performances').doc(performanceId);
```

Comment:
Is it possible for us to use onSnapshot rather than get() here? Get is a one time request, onSnapshot registers a callback to continually update state if the performance changes. 

Also: The collection.doc.collection.doc... call can be summarized. Take a look at the collectionName and the db call below. It summarizes all the collections and docs into one path and then expands on that.